docs/usage/key.rst

```diff
@@ -35,9 +35,11 @@ Fully automated using environment variables:
 
 ::
 
-    $ BORG_NEW_PASSPHRASE=old borg init -e=repokey repo
+    $ BORG_PASSPHRASE='old'
+    $ borg init -e=repokey repo
```

Comment:
i bet this does not work (without "export" this does not stay in the environment).

but stuff like `KEY=value command` works and temporarily puts this into the env of the command process.